substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/YoungGeneration.java

```diff
@@ -86,18 +120,60 @@ public final Space getSpace() {
         return space;
     }
 
+    private final SurvivorSpace getSurvivorToSpaceAt(int index) {
```

Comment:
I'm generally fine to not have accessor methods for private or project-private fields, to keep the code shorter. In this particular case I think the assertion is also not overly useful so it can be dropped too.

Reply:
done